src/app/src/components/ClaimedFacilitiesDetails.jsx

```diff
@@ -1,9 +1,543 @@
+                    onChange={updateFacilityAddress}
+                    disabled={updating}
+                />
+                <InputSection
+                    label="Country"
+                    value={data.facility_country}
+                    onChange={updateFacilityCountry}
+                    disabled={updating}
+                    isSelect
+                    selectOptions={countryOptions}
+                />
+                <InputSection
+                    label="Phone Number"
+                    value={data.facility_phone_number}
+                    onChange={updateFacilityPhone}
+                    disabled={updating}
+                    hasSwitch
+                    switchValue={data.facility_phone_number_publicly_visible}
+                    onSwitchChange={updateFacilityPhoneVisibility}
+                />
+                <InputSection
+                    label="Website"
+                    value={data.facility_website}
+                    onChange={updateFacilityWebsite}
+                    disabled={updating}
+                />
+                <InputSection
+                    label="Description"
+                    value={data.facility_description}
+                    multiline
+                    onChange={updateFacilityDescription}
+                    disabled={updating}
+                />
+                <InputSection
+                    label="Minimum order quantity"
+                    value={data.facility_minimum_order_quantity}
+                    onChange={updateFacilityMinimumOrder}
+                    disabled={updating}
+                />
+                <InputSection
+                    label="Average lead time"
+                    value={data.facility_average_lead_time}
+                    onChange={updateFacilityAverageLeadTime}
+                    disabled={updating}
+                />
+                <Typography
+                    variant="title"
+                    style={claimedFacilitiesDetailsStyles.headingStyles}
+                >
+                    Point of contact
+                    <span
+                        style={
+                            claimedFacilitiesDetailsStyles.switchSectionStyles
+                        }
+                    >
+                        <Switch
+                            color="primary"
+                            onChange={updateContactVisibility}
+                            checked={data.point_of_contact_publicly_visible}
+                        />
+                        Publicly visible
+                    </span>
+                </Typography>
+                <InputSection
+                    label="Contact person name"
+                    value={data.point_of_contact_person_name}
+                    onChange={updateContactPerson}
+                    disabled={updating}
+                />
+                <InputSection
+                    label="Email"
```

Comment:
Consider creating a follow up issue to add email address validation to this field.

Reply:
This should be trivial to incorporate here, so I can just add it in this PR. I think we want to validate `isEmpty(email) || isEmail(email)` for this field, and probably disable the save button concurrently.

Reply:
Added in 9c3296bf669dbadd3cc657cea27db06b0872f3ff